docs: add PostgreSQL sync schema (#661)
bytebase/bytebase.com: content/docs/change-database/synchronize-schema.md

<hint-block type="warning">

This feature is in beta and only available for MySQL and PostgreSQL.

> The feature PostgreSQL Synchronize Schema now only supports to synchronize `Schema`, `Table`, `Index`, `Constraint` and `Sequence` objects.

Reviewer: ```suggestion
Synchronize Schema for PostgreSQL currently supports synchronizing `Schema`, `Table`, `Index`, `Constraint`, and `Sequence` objects.
```

Reviewer: How about MySQL?
